Weather/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,56 @@
+/* 
+  Localizable.strings
+  Weather
+
+  Created by Oleshkevich Artem on 7.09.22.
+  
+*/
+"Today" = "Today";
+"Temperature unit" = "°C";
```

Comment:
It's better to name your keys using camelCase: `today`, `temperatureUnit` and so on